ports/atmel-samd/common-hal/canio/Listener.h

```diff
@@ -47,7 +47,7 @@ typedef struct {
 void common_hal_canio_listener_construct(canio_listener_obj_t *self, canio_can_obj_t *can, size_t nmatch, canio_match_obj_t **matches, float timeout);
 void common_hal_canio_listener_check_for_deinit(canio_listener_obj_t *self);
 void common_hal_canio_listener_deinit(canio_listener_obj_t *self);
-bool common_hal_canio_listener_readinto(canio_listener_obj_t *self, canio_message_obj_t *message);
+bool common_hal_canio_listener_receiveinto(canio_listener_obj_t *self, canio_message_obj_t *message);
```

Comment:
No need to have these functions here because they are in `shared-bindings/canio/Listener.h`.